docs/pages/cli-docs.mdx

```diff
@@ -532,22 +532,37 @@ registered devices using [`tsh mfa ls`](cli-docs.mdx#tsh-mfa-ls).
 
 ## tctl
 
-`tctl` is an admin CLI tool used to administer a Teleport cluster. It connects
-to an Auth Server, meaning that it must be run on the same host with the role
-`auth` . If you run `tctl` on a non- `auth` node it will show an error.
+`tctl` is an admin CLI tool used to administer a Teleport cluster. 
```

Comment:
Nit: Bit redundant. Can we just say CLI tool? Or further specify that it allows one to administer other admin features/tools?